src/main/java/net/dries007/tfc/ConfigTFC.java

```diff
@@ -186,6 +186,10 @@ public static void onConfigChangedEvent(ConfigChangedEvent.OnConfigChangedEvent
         @Config.Comment("Should the prospectors pick output to the actionbar? (the space just above the hotbar)")
         @Config.LangKey("config." + MOD_ID + ".client.propickOutputToActionBar")
         public boolean propickOutputToActionBar = true;
+
+        @Config.Comment("Show extra player health using additional health bar overlays? (for Absorption or increased max health)")
+        @Config.LangKey("config." + MOD_ID + ".client.showExtraHealthWithOverlays")
+        public boolean showExtraHealthWithOverlays = true;
```

Comment:
As of b7c9c2e6a105d91e56d6c359925ea2099113538c I'm going to be picky about this: Please add an `en_us.lang` entry with the lang key and tooltip of any config values you add. See the existing ones for style - the name should be a readable version of the variable name and the tooltip should match the comment.

Reply:
Lang file updated to match.